### PR TITLE
Updated composer package versions for 2.4

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) updates Composer package versions to correct 2.4.x sample data upgrade instructions.

Only lines 35 and 45 in the `src/_includes/install/sampledata/sample-data-rc1-cli-24.md` file were changed.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/comp-mgr/cli/cli-rc1-samp.html

whatsnew
Updated Composer package versions to correct 2.4.x [sample data upgrade instructions](https://devdocs.magento.com/guides/v2.4/comp-mgr/cli/cli-rc1-samp.html).

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]

File: src/_includes/install/sampledata/sample-data-rc1-cli-24.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+   ```
+
+1. If prompted, enter your [authentication keys].
+1. Wait for dependencies to update.
+
+### Finish your upgrade
+
+After you have reset file system permissions:
+
+1. If you haven't done so already, log in to your Magento server as, or switch to, the Magento file system owner.
```
Comment (dobooth) [inline src/_includes/install/sampledata/sample-data-rc1-cli-24.md:66]: Contraction.

File: src/_includes/install/sampledata/sample-data-rc1-cli-24.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   A sample command follows:
+
+   ```bash
+   rm -rf var/cache/* var/page_cache/* var/generation/*
+   ```
+
+1. Upgrade Magento:
+
+   ```bash
+   php bin/magento setup:upgrade
```
Comment (dobooth) [inline src/_includes/install/sampledata/sample-data-rc1-cli-24.md:79]: I thought we were removing `php` from bin/magneto commands.

Comment (dobooth) [reply src/_includes/install/sampledata/sample-data-rc1-cli-24.md:79]: hmm...won't let me comment on the cli-rc1-samp.md below. Do you need a space between %include on L8?

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/_includes/install/sampledata/sample-data-rc1-cli-24.md (hunk 2 above)
Comment (jeff-matthews) [reply src/_includes/install/sampledata/sample-data-rc1-cli-24.md:79]: I just broke the symlink and created a new file. I only modified lines 35 & 45 as mentioned in the PR description.

**Review by dobooth — APPROVED** [review]

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**